bitbot/links.go

```diff
@@ -21,7 +21,7 @@ var URLReaderTrigger = NamedTrigger{
 		resp := lookupPageTitle(m.Content)
 		if resp != "" {
 			irc.Reply(m, lookupPageTitle(m.Content))
-			if len(resp) > 70 {
+			if len(m.Content) > 70 {
```

Comment:
this could still trigger whenever someone adds commentary around a URL but eh, good enough. It fixes the problem of URLs breaking because of a wrap in the terminal